app/build.gradle

```diff
@@ -105,6 +105,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
     implementation 'com.google.android.material:material:1.8.0'
+    implementation project(path: ':external:calendar')
```

Comment:
why do you need this?

Reply:
aha, I have no idea why it changes. I did not make any change over there. :((